scripts/lib/clean_unused_caches.py

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/env python3
+import argparse
 import os
 import sys
+from typing import Union
 
 ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 sys.path.append(ZULIP_PATH)
 from scripts.lib import clean_emoji_cache, clean_node_cache, clean_venv_cache, clean_yarn_cache
 from scripts.lib.zulip_tools import parse_cache_script_args
 
 
-def main() -> None:
-    args = parse_cache_script_args("This script cleans unused Zulip caches.")
+def main(args: Union[argparse.Namespace, None] = None) -> None:
+    if not args:
+        args = parse_cache_script_args("This script cleans unused Zulip caches.")
```

Comment:
Can this just be moved to the `main()` call below?

Reply:
Yeah, That would be better.
Done!